deployment/src/main/java/io/quarkiverse/fury/deployment/FuryProcessor.java

```diff
@@ -86,20 +83,29 @@ public void registerResteasyIntegration(Capabilities capabilities,
             BuildProducer<ResteasyJaxrsProviderBuildItem> resteasyJaxrsProviderBuildItemBuildProducer,
             BuildProducer<MessageBodyReaderBuildItem> additionalReaders,
             BuildProducer<MessageBodyWriterBuildItem> additionalWriters) {
-        if (capabilities.isPresent(Capability.RESTEASY)) {
+        if (capabilities.isPresent(Capability.RESTEASY) || capabilities.isPresent(Capability.RESTEASY_CLIENT)) {
             resteasyJaxrsProviderBuildItemBuildProducer
                     .produce(new ResteasyJaxrsProviderBuildItem(FurySerializer.class.getName()));
```

Comment:
I am not sure if this is enough for RESTEasy Classic. You probable also have to use `META-INF/services/jakarta.ws.rs.ext.Providers` like RESTEasy does in the `resteasy-json-binding-provider` module for example